docker/README.md

```diff
@@ -20,7 +20,7 @@ $ docker run --env STORAGE_TYPE=cassandra --env CASSANDRA_CONTACT_POINTS=host1,h
 To process spans since midnight every hour, and all spans each day, change the entrypoint to cron.
```

Comment:
dunno how we managed to say this and not actually show changing the entrypoint :P